### PR TITLE
Story/ccp 118

### PR DESCRIPTION
**Pull request by mkarthik90**
[CCP-118] Resolved Merge Conflicts

### CODE REVIEW

File: Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/LoginController.java [hunk 1]
```diff
@@ -19,11 +20,10 @@
 	 * 
 	 */
 	@RequestMapping(value = "/login", method = RequestMethod.GET)
-	public String login() {
+	public String login(@ModelAttribute("conceptSearchBean")ConceptSearchBean conceptSearchBean) {
+        return "welcome";
```
Comment (jdamerow) [inline Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/LoginController.java]: what's up with the indentation?

**Comment by jdamerow** [comment]
please resolve conflicts.

**Comment by mkarthik90** [comment]
@jdamerow  I have resolved merge conflicts and did the formatting changes as suggested by you.

**Comment by jdamerow** [comment]
ok to test

**Comment by jdamerow** [comment]
The drop down now shows VERB, NOUN, etc. It should show Verb, Noun, etc. as before.
image

**Comment by mkarthik90** [comment]
@jdamerow  I have made the changes for displaying Noun, Verb, Adjective.. The issue was because previously before the change the drop down was hardcoded in the jsp page. And now since we have moved the changes to model attribtute and fetched the data from enum I did not make the style changes. I have made the change in the model attribute and not in the enum. If we change in the enum it might cause issues while fetching the data, since instead of passing NOUN to the db we will be passing Noun so I made the change in model attribute.

**Comment by jdamerow** [comment]
retest this please

**Pull request merged**